docs/examples.rst

```diff
@@ -105,89 +112,93 @@ Grab screenshot or image from the clipboard
 Grab frames from your webcam
 ----------------------------
 
-Use the special ``<video0>`` uri to read frames from your webcam (via
-the ffmpeg plugin). You can replace the zero with another index in case
-you have multiple cameras attached. You need to ``pip install imageio-ffmpeg``
-in order to use this plugin.
+.. note::
+    For this to work, you need to install the ffmpeg backend::
```

Comment:
or pyav? ;)

Reply:
I would actually go with `imageio-ffmpeg` for now and then update once pyav is merged and had some time to prove itself.